config/packages/easy_admin.yaml

```diff
@@ -0,0 +1,6 @@
+#easy_admin:
```

Comment:
Šiam namų darbui easy admin bundle nereikėjo.

Darau prielaidą, kad nepalikinėti šiukšlių išmoksi ir be papildomo skatinimo.